### PR TITLE
Added a tool that gets the list of failed clusters from the log-serve…

### PR DESCRIPTION
**Pull request by ronniel1**
…r, and creates a Jira ticket for the triaging

See tickets created: https://issues.redhat.com/issues/?jql=component%20%3D%20%22Assisted-installer%20Triage%22

### CODE REVIEW

File: tools/create_triage_tickets.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    parser.add_argument("--netrc", default=DEFAULT_NETRC_FILE, help="netrc file")
+    parser.add_argument("-v", "--verbose", action="store_true", help="Output verbose logging")
+    args = parser.parse_args()
+
+    logging.basicConfig(level=logging.WARN, format='%(levelname)-10s %(message)s')
+    logger = logging.getLogger(__name__)
+    logging.getLogger("__main__").setLevel(logging.INFO)
+
+    if args.verbose:
+        is_verbose = True
```
Comment (YuviGold) [inline tools/create_triage_tickets.py:143]: not used

**Review by YuviGold — APPROVED** [review]
LGTM

**Review by filanov — APPROVED** [review]

**Comment by romfreiman** [comment]
@ronniel1 maybe it's better to aggregate it with @YuviGold work on logs download? Or it's unrelated?

**Comment by ronniel1** [comment]
Moved this to assisted-test-infra: https://github.com/openshift/assisted-test-infra/pull/193

**Pull request closed without merging**